runtime/src/accounts_db.rs

```diff
@@ -2743,13 +3607,21 @@ impl AccountsDB {
         }
     }
 
+    pub fn store_cached(&self, slot: Slot, accounts: &[(&Pubkey, &Account)]) {
+        self.do_store(slot, accounts, self.caching_enabled);
+    }
+
     /// Store the account update.
-    pub fn store(&self, slot: Slot, accounts: &[(&Pubkey, &Account)]) {
+    pub fn store_uncached(&self, slot: Slot, accounts: &[(&Pubkey, &Account)]) {
         // If all transactions in a batch are errored,
         // it's possible to get a store with no accounts.
         if accounts.is_empty() {
             return;
         }
+        self.do_store(slot, accounts, false);
+    }
+
+    fn do_store(&self, slot: Slot, accounts: &[(&Pubkey, &Account)], use_cache: bool) {
```

Comment:
now that we have `use_cache` and `self.caching_enabled`, this is getting confusing. how about renaming this to `is_cached_storing` or `to_cache` or `store_to_cache` or `write_to_cache` or `cache_hint`?

Pick any one. :)

Reply:
changed to `is_cached_store`!